packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/agents/ovhPabx/ovh-pabx.html

```diff
@@ -61,47 +73,35 @@ <h1 data-translate="telephony_alias_configuration_agents_title"></h1>
                             data-on-choice-validated="AgentsOvhPabxCtrl.onChooseServicePopover"
                             data-on-choice-cancel=""
                         >
-                            <span class="ovh-font ovh-font-search"></span>
+                            <span class="oui-icon oui-icon-search"></span>
```

Comment:
```suggestion
                            <span class="oui-icon oui-icon-search" aria-hidden="true"></span>
```
Maybe some text description could be added for `sr-only` ?